### PR TITLE
feat(capture-sdk): Disable flash by default

### PR DESCRIPTION
**Pull request by abolfazlimahdi**
BSDK-95

### CODE REVIEW

File: bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/Configuration.kt [hunk 1]
```diff
@@ -89,7 +89,7 @@ data class CaptureConfiguration(
     /**
      * Set whether the camera flash is on or off by default.
```
Comment (a-szotyori) [inline bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/Configuration.kt:90]: [Suggestion] What do you think of repharsing it to:
```suggestion
     * Set whether the camera flash should be on or off when the SDK starts. The flash is off by default.
```

File: bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/Configuration.kt [hunk 2]
```diff
@@ -89,7 +89,7 @@ data class CaptureConfiguration(
...
      */
-    val flashOnByDefault: Boolean = true,
+    val flashOnByDefault: Boolean = false,
```
[the remark this answers is not in the document]
Comment (abolfazlimahdi) [reply bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/Configuration.kt:92]: Done! Thanks for noticing :)

File: bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/Configuration.kt (hunk 1 above)
Comment (abolfazlimahdi) [reply bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/Configuration.kt]: Done! Also changed for capture docs :-)

**Review by a-szotyori — APPROVED** [review]
Nice!

**Review by a-szotyori — APPROVED** [review]
👏

**Pull request merged**